.gitignore

```diff
@@ -57,7 +57,3 @@ logs/
 
 # Python cache
 __pycache__
-
-# Cargo files
-Cargo.lock
-Cargo.toml
```

Comment:
Why do we commit cargo locks here but not in Protocol or TypeQL?

Reply:
TypeDB is a binary, and it's accepted practice in Rust community to provide lockfiles. Libraries cannot rely on lockfiles though.